### ISSUE TITLE
mebeh too much jayhess?

### ISSUE THREAD

**Issue by TheAlphaNerd** [wafflejs/wafflejs.github.io#27]
The current site is built with react. I had a blast going through the source and it is super elegant.

One problem, if you visit the site with jayhess turned off you get a big ole blank screen. WOMP WOMP

I know that react has some ability to do server side static rendering, and since the site does not have dynamic content it may make sense to statically generate the payload.

I tried visiting the site with lynx and was served the markdown :S.  I do not believe that the jayhess client rendering will effect screen readers, but I have not checked yet.

TLDR; maybe statically generate payload?

**Comment by visnup** [comment]
actually, react is coming, but not in at all yet. it's just marked + markdown + craziness. I could maybe throw all the markdown into a no script tag actually... ooh.
